thumbnails/docs/getting-started.md

```diff
@@ -0,0 +1,290 @@
+---
+title: "Getting Started"
+date: 2018-05-02T00:00:00+00:00
+weight: 20
+geekdocRepo: https://github.com/owncloud/ocis-thumbnails
+geekdocEditPath: edit/master/docs
+geekdocFilePath: getting-started.md
+---
+
+{{< toc >}}
+
+## Installation
+
+So far we are offering two different variants for the installation. You can choose between [Docker](https://www.docker.com/) or pre-built binaries which are stored on our download mirrors and GitHub releases. Maybe we will also provide system packages for the major distributions later if we see the need for it.
+
+### Docker
+
+TBD
+
+### Binaries
+
+TBD
+
+## Configuration
```

Comment:
Codacy found an issue: [[heading-style] Headings should use setext](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)